src/axelrod_dojo/utils.py

```diff
@@ -11,17 +10,20 @@
 ## Output Evolutionary Algorithm results
 
 class Outputer(object):
-    def __init__(self, filename, mode='w'):
-        self.output = open(filename, mode)
-        self.writer = csv.writer(self.output)
+    def __init__(self, filename, mode='a'):
+        self.file = filename
+        self.mode = mode
 
-    def write(self, row):
-        self.writer.writerow(row)
-        self.output.flush()
-        os.fsync(self.output.fileno())
+    def write_row(self, row):
+        with open(self.file, self.mode, newline='') as file_writer:
```

Comment:
Are you sure that you want no newline here?

Reply:
it makes the csv cleaner on output, I can change it back if needed. The solve was from [this](https://stackoverflow.com/questions/3191528/csv-in-python-adding-an-extra-carriage-return) thread on having too many carriage returns on windows.